### PR TITLE
Increase code coverage to 50%

### PR DESCRIPTION
**Pull request by andriisoldatenko**
Increase code coverage up to 50% in total.

fix astronomer/issues#1329

### CODE REVIEW

File: airflow/airflow_test.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	assert.NoError(t, err)
+
+	exist, err := fileutil.Exists(filepath.Join(tmpDir, "requirements.txt"))
+
+	assert.NoError(t, err)
+	assert.True(t, exist)
+
+}
+
+
```
Comment (adam2k) [inline airflow/airflow_test.go:65]: This is a nit pick, but when I linted `gofmt` removed this line.

**Comment by adam2k** [comment]
I didn't notice this was [WIP].  I'll wait to approve until you have all the changes in.

**Comment by andriisoldatenko** [comment]
@schnie @adam2k PTAL

**Comment by andriisoldatenko** [comment]
>Maybe we should consider moving those really long okResponse strings into separate test files similarly to test tables to make for shorter testing files?
in general it's a good idea but in practise we loose readability especially when we need to demonstrate the repose from houston. E.g. two similar tests have 1 line diff in okResponse from houston.

File: Makefile [hunk 2]
```diff
@@ -29,7 +29,7 @@ cover:
 
 format:
 	@echo "--> Running go fmt"
-	@go fmt $(GOFILES)
+	@go fmt ./...
```
Comment (adam2k) [inline Makefile:32]: 🚀👍

**Comment by adam2k** [comment]
> > Maybe we should consider moving those really long okResponse strings into separate test files similarly to test tables to make for shorter testing files?
> 
> in general it's a good idea but in practise we loose readability especially when we need to demonstrate the repose from houston. E.g. two similar tests have 1 line diff in okResponse from houston.

This is OK for now.  We can come up with something to improve this later.

**Comment by andriisoldatenko** [comment]
@schnie can i merge this?

**Pull request merged**